metering/metering-about-installing-metering.adoc

```diff
@@ -7,6 +7,13 @@ toc::[]
 
 Before installing metering into your cluster, review the following sections.
 
+[WARNING]
+====
+Installing Metering on an Openshift FIPS-enabled cluster fails during the execution of the `images/metering-ansible-operator/roles/meteringconfig/tasks/configure_root_ca.yml` task file.
```

Comment:
For context, we enable TLS/authentication between the Metering pod components by default. In the case where using this default behavior (i.e. not explicitly disabling TLS via `tls.enabled: false`), will execution will attempting to generate TLS certificates.

I think we may want to stray away from specifying the path to the task file as the BZ should have that information, and just point out that deploying Metering will fail on a FIPS-enabled cluster when TLS is enabled.

Reply:
I think the wording on this can be improved, but I was thinking of something along the lines of:

Installing Metering on an Openshift FIPS-enabled cluster with TLS enabled will fail while attempting to generate TLS certificates for the user.